### DISCUSSION TITLE
protected static $logName = 'system';

### DISCUSSION THREAD

**Discussion by rodrigompais** [spatie/laravel-activitylog#783] [Q&A]
Guys, how do I put the user name logged in the field
protected static $logName = 'system';

**Comment by Gummibeer** [comment]
Hey,

you could use the `tap()` or `tapActivity()` methods or an observer.

But I don't recommend to do so because it's a duplication of data and misuse of the column. If you want to persist the username on top of the relationship you should use a new column or the properties JSON column.

**Reply by rodrigompais** [reply]
Hello, how are you? I didn't understand where to put it, my code is like this.
```
class Status extends Model
{
    use LogsActivity;

    protected static $logAttributes = ['*'];

    protected static $logOnlyDirty = true;

    protected static $logName = 'system';

    protected $fillable = ['name','uuid'];

    //protected $table = 'statuses';
    
    public function produto()
    {
        return $this->hasMany(Produto::class);
    }
}
```

**Comment by Gummibeer** [comment]
https://spatie.be/docs/laravel-activitylog/v3/advanced-usage/logging-model-events#tap-activity-before-logged-from-event

Quick link from mobile - let me know if you need more.
